src/run/runner/valgrind/measure.rs

```diff
@@ -0,0 +1,99 @@
+use crate::prelude::*;
+use crate::run::runner::helpers::env::BASE_INJECTED_ENV;
+use crate::run::runner::helpers::get_bench_command::get_bench_command;
+use crate::run::runner::helpers::run_command_with_log_pipe::run_command_with_log_pipe;
+use crate::run::runner::valgrind::helpers::ignored_objects_path::get_objects_path_to_ignore;
+use crate::run::runner::valgrind::helpers::introspected_node::setup_introspected_node;
+use crate::run::{config::Config, instruments::mongo_tracer::MongoTracer};
+use lazy_static::lazy_static;
+use std::env;
+use std::fs::canonicalize;
+use std::path::Path;
+use std::{env::consts::ARCH, process::Command};
+
+lazy_static! {
+    static ref VALGRIND_BASE_ARGS: Vec<String> = {
+        let mut args = vec![];
+        args.extend(
+            [
+                "-q",
+                "--tool=callgrind",
+                "--trace-children=yes",
+                "--cache-sim=yes",
+                "--I1=32768,8,64",
+                "--D1=32768,8,64",
+                "--LL=8388608,16,64",
+                "--instr-atstart=no",
+                "--collect-systime=nsec",
+                "--compress-strings=no",
+                "--combine-dumps=yes",
+                "--dump-line=no",
+            ]
+            .iter()
+            .map(|x| x.to_string()),
+        );
+        let children_skip_patterns = ["*esbuild"];
+        args.push(format!(
+            "--trace-children-skip={}",
+            children_skip_patterns.join(",")
+        ));
+        args
+    };
+}
+
+pub const VALGRIND_EXECUTION_TARGET: &str = "valgrind::execution";
+
+pub fn measure(
+    config: &Config,
+    profile_folder: &Path,
+    mongo_tracer: &Option<MongoTracer>,
+) -> Result<()> {
+    // Create the command
+    let mut cmd = Command::new("setarch");
+    cmd.arg(ARCH).arg("-R");
+    // Configure the environment
+    cmd.envs(BASE_INJECTED_ENV.iter()).env(
+        "PATH",
+        format!(
+            "{}:{}",
+            setup_introspected_node()
```

Comment:
```suggestion
            setup_introspected_nodejs()
```
Maybe a little bit more explicit